sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Properties/AssemblyInfo.cs

```diff
@@ -12,7 +12,7 @@
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
 [assembly: AssemblyVersion("0.20.0.0")]
-[assembly: AssemblyFileVersion("0.20.6.0")]
+[assembly: AssemblyFileVersion("0.20.8.0")]
```

Comment:
Not sure why 0.20.7 was skipped 🤔 ; besides please also update the version in .csproj

Reply:
Addressed.